### PR TITLE
Add styled footer to website:

### PR DESCRIPTION
**Pull request by ericcheatham**
**Issue being addressed:**
Need to update footer

**How does this PR address it?**
- Installs `Roboto Condensed` font
- Changes colors in the footer
- Adds social links

**What needs to be documented once your changes are merged?**
n/a

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,8 +5,11 @@
   "repository": "https://github.com/dsmHack/pi-515",
   "license": "UNLICENSED",
   "dependencies": {
-    "codemirror": "^5.46.0",
+    "@fortawesome/fontawesome-svg-core": "^1.2.27",
+    "@fortawesome/free-brands-svg-icons": "^5.12.1",
+    "@fortawesome/react-fontawesome": "^0.1.9",
     "bulma": "^0.8.0",
+    "codemirror": "^5.46.0",
```
Comment (nickolaswilson-wf) [inline package.json:12]: ABC 👌👌👌

File: src/components/Footer.js [hunk 2]
```diff
@@ -1,110 +1,31 @@
...
+                <a title="linkedin" href="https://www.linkedin.com/company/pi-515/">
+                  <FontAwesomeIcon icon={faLinkedin} color="black" size="lg"/>
                 </a>
-                <a title="vimeo" href="https://vimeo.com">
-                  <img
-                    src={vimeo}
-                    alt="Vimeo"
-                    style={{ width: '1em', height: '1em' }}
-                  />
+                <a title="instagram" href="https://instagram.com">
```
Comment (nickolaswilson-wf) [inline src/components/Footer.js]: I'm not sure this is needed as they don't have an Instagram as far as I know.

**Review by nickolaswilson-wf — CHANGES_REQUESTED** [review]

File: src/components/Footer.js (hunk 2 above)
Comment (ericcheatham) [reply src/components/Footer.js]: They do. She created it last night.

**Review by nickolaswilson-wf — APPROVED** [review]
+1 🔥

**Pull request merged**